CHANGELOG.md

```diff
@@ -180,6 +181,17 @@
 - Fixed an issue for tracing PDK where sample rate does not work.
   [#10485](https://github.com/Kong/kong/pull/10485)
 
+### Breaking Changes
+
+#### Plugins
+
+- **http-log, statsd, opentelemetry, datadog**: The queueing system
+  has been reworked, causing some plugin parameters to not
+  function as expected anymorechem and new parameters to be added.  The
+  module `kong.tools.batch_queue` has been renamed to `kong.tools.batch`
+  in the process.
```

Comment:
```suggestion
  in the process and requires you to do foo bar. Please note that the API for the old and new Lua module are different.
```

Reply:
ping @hanshuebner

Reply:
The CHANGELOG entry will be updated with a link to the blog post describing the queueing changes once that is done.